ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-release-4.8.yaml

```diff
@@ -3,35 +3,15 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
-  ocp_builder_golang-1.14:
-    name: builder
-    namespace: ocp
-    tag: golang-1.14
-  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.15-openshift-4.6
   ocp_builder_rhel-8-golang-1.15-openshift-4.7:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.15-openshift-4.7
-  ocp_builder_rhel-8-golang-openshift-4.6:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
-  openshift_release_golang-1.14:
-    name: release
-    namespace: openshift
-    tag: golang-1.14
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
```

Comment:
rehearsals are failing due to 4.8 not being built yet. Lets leave this for now and regen it after 4.7 is frozen?

Reply:
Leaving 4.8 at 1.13 seems like it might surprise us later.  I am ok landing this without green rehearsals.  I am also ok leaving 4.8 alone, getting green rehearsals, and then making the same changes to 4.8, and landing this PR.

Reply:
@vrutkovs @wking Can we go ahead and land this "as is" then? This is holding me up from pushing up a CVO PR I'd like to get off my plate.

Reply:
Lets push a commit which doesn't change 4.8, check that rehearsals are valid and then re-introduce 4.8 change